build.sbt

```diff
@@ -107,7 +107,7 @@ lazy val docs = project
   )
   .settings(commonSettings)
   .settings(
-    crossScalaVersions := Seq(scala212, scala211),
+    crossScalaVersions := Seq(scala212, scala211, scala213),
```

Comment:
Fixing the build surfaced some issues with the `docs` project setup.

Reply:
I've reviewed `build.sbt` and this change looks really weird because `commonSettings` already sets `crossScalaVersions := Seq(scala212, scala211, scala213)`